### PR TITLE
Adds testkit to use a gRPC client in Lagom tests

### PR DESCRIPTION
**Pull request by ignasi35**
Fixes https://github.com/lagom/lagom/issues/1615

- [x] add the testkit helpers
- [x] test 
- [x] document

### CODE REVIEW

**Comment by dwijnand** [comment]
... and maybe not in it's current form, but this repo intended to subsume the play generation bits still left over in akka/akka-grpc.

**Comment by ignasi35** [comment]
ATM, a Lagom user that wants to use gRPC has a project with the `"lagom-sbt-plugin"` and `"sbt-akka-grpc"` plugins but once the generators for Play in `akka-grpc` are moved these dependencies will change to something like: `"lagom-sbt-plugin"` and `"sbt-play-grpc"`.

Creating a new `lagom-grpc` for 20 lines of code to be used on tests seems overkill. Also, considering the trend to converge I think we can consider `play-grpc` as the icing you put on top of Play or Lagom to use gRPC. Does that make sense?

**Comment by dwijnand** [comment]
If there's no cycle then we're groovy.

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ jobs:
   include:
     - script:
       - bin/scalafmt --test
+    - script:
```
Comment (dwijnand) [inline .travis.yml]: What's the motivation?

Comment (ignasi35) [reply .travis.yml]: In a previous commit, the `bin/scalafmt` command failed without failing the job immediately. Then, the `sbt test...` command succeeded so the Travis job failed but the failure was hidden halfway the output.

This way makes it very clear what the issue really is.

To be honest, I'd event split 

```yaml
  - script:
    - sbt test headerCheck test:headerCheck` 
```

into

```yaml
  - script:
    - sbt headerCheck test:headerCheck` 
  - script:
    - sbt test
```

Comment (dwijnand) [reply .travis.yml]: In that case, there's a better solution.

The problem with splitting into multiple jobs is that it causes overhead, so it's best to minimise.

To have this fail fast and obviously I suggest:

```yaml
  - script:
    - bin/scalafmt --test || travis_terminate 1
    - sbt headerCheck test:headerCheck test
```

Comment (ignasi35) [reply .travis.yml]: Hmm, but then a silly formatting issue masks other problems in tests or header checks.

I want all the info asap and easily accessible. 

Ok, I'll revert this and we can discuss on a separate PR.

File: lagom-interop-test-scala/src/test/scala/com/example/hello/impl/HelloServiceSpec.scala [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+    // #unmanaged-client
+    "say hello over gRPC (unmnanaged client)" in {
+      grpcClient
+        .sayHello(HelloRequest("Alice"))
+        .map {
+          _.message should be("Hi Alice! (gRPC)")
+        }
+    }
+    // #unmanaged-client
```
Comment (ignasi35) [inline lagom-interop-test-scala/src/test/scala/com/example/hello/impl/HelloServiceSpec.scala]: The snippet labelled `#unmanaged-client` is the sum of two bits of code: (1) the creation and `close`, and (2) the use of the instance.

**Pull request merged**